workspace/python/ds-pack/2.1.0-base-python-3.10.13/codeserver-proxy/jupyter_codeserver_proxy/__init__.py

```diff
@@ -0,0 +1,28 @@
+import os
+import subprocess
+
+bash_command = "code-server --install-extension cweijan.vscode-database-client2@6.6.3"
+
+def setup_codeserver():
+  _install_vscode_extension()
```

Comment:
Any reason for installing it here? We can install it from `Dockerfile` as well just like other Python packages?

Reply:
Hey yes. It actually doesn't work when we install it in the `Dockerfile` 😅